WordPress/gutenberg
# Issue 331/mention support on @ keypress

## Description


Change the rich text component in mobile in order to intercept any random keypress.

## How has this been tested?




This can be tested by running: `npm run native ios` or `npm run native android` after the demo app starts, follow these steps:

 - Add a Paragraph block
 - Tap the @ on the keyboard
 - Check if a mention to `matt` is added
 - Check if the mention is only inserted if the @ is inserted with a space before or at the start of the paragraph.
 - Check if not mention is inserted on the title of the post.

Related PR: GB-Mobile: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2424

## Screenshots 

## Types of changes





## Checklist:
